.github/workflows/ci.yml

```diff
@@ -110,6 +110,17 @@ jobs:
     - name: Compile additional subprojects
       if: ${{ ((startsWith(matrix.scala, '2.12.')) || (startsWith(matrix.scala, '2.13.'))) && (matrix.java == '11') }}
       run: sbt ++${{ matrix.scala }} examplesCore/compile examplesJpl/compile examplesSlf4j2Bridge/compile examplesSlf4jLogback/compile examplesSlf4j2Logback/compile examplesSlf4j2Log4j/compile benchmarks/compile
+  ci:
+    name: CI
```

Comment:
it does not seem to be ok, (based on build), this name should not probably be here